src/DynamicData.Tests/Utilities/ObservableSpy.cs

```diff
@@ -125,6 +126,28 @@ private static class NativeMethods
     private static Func<IChangeSet<T>, string> CreateListChangeSetFormatter<T>(Func<T, string> formatter) where T : notnull =>
         cs => "[List Change Set]" + ChangeSetEntrySpacing + string.Join(ChangeSetEntrySpacing, cs.Select((change, n) => $"#{n} {FormatChange(formatter, change)}"));
 
+    public static IObservable<T> TestSpy<T>(this IObservable<T> source,
+                                                                ITestOutputHelper testOutputHelper, string? opName = null,
```

Comment:
This indentation makes my eyes twitch. :P

Reply:
I'll flatten them out like all the others.